tests/functional/pages/surveys/question.page.js

```diff
@@ -33,6 +33,11 @@ class QuestionPage {
     return browser.click('a[id="top-previous"]')
   }
 
+  next() {
```

Comment:
Although the fix is for UKIS, it is valid for all surveys, so a shorter one would make sense